airflow/models/xcom.py

```diff
@@ -772,6 +772,8 @@ def _get_bound_query(self) -> Generator[Query, None, None]:
             yield self._query
             return
 
+        if Session is None:
+            raise
```

Comment:
```suggestion
        if getattr(setting, "Session", None) is None:
            raise RuntimeError("Session must be set before!")
```

This matches the pattern elsewhere; if this is called before `settings` is configured, the `Session` variable may actually not exist at all (instead of being `None`).